server/src/main/java/org/bonitasoft/console/common/server/page/BDMClientDependenciesResolver.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (C) 2015 Bonitasoft S.A.
+ * Bonitasoft, 32 rue Gustave Eiffel - 38000 Grenoble
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2.0 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.bonitasoft.console.common.server.page;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.apache.commons.io.FileUtils;
+import org.bonitasoft.console.common.server.preferences.constants.WebBonitaConstantsUtils;
+import org.bonitasoft.engine.api.TenantAPIAccessor;
+import org.bonitasoft.engine.api.TenantAdministrationAPI;
+import org.bonitasoft.engine.business.data.BusinessDataRepositoryException;
+import org.bonitasoft.engine.exception.BonitaHomeNotSetException;
+import org.bonitasoft.engine.exception.ServerAPIException;
+import org.bonitasoft.engine.exception.UnknownAPITypeException;
+import org.bonitasoft.engine.io.IOUtil;
+import org.bonitasoft.engine.session.APISession;
+
+public class BDMClientDependenciesResolver {
+
+    private final APISession session;
+
+    private static Logger LOGGER = Logger.getLogger(BDMClientDependenciesResolver.class.getName());
+
+    private static final String BDM_MODEL_JAR_NAME = "bdm-model.jar";
+    private static final String BDM_DAO_JAR_NAME = "bdm-dao.jar";
+    private static final String JAVASSIST_JAR_NAME = "javassist-3.18.1-GA.jar";
```

Comment:
is javassist really necessary ?
And shouldn't we take everything jar that is in the zip instead of naming them ?